packages/angularjs-extensions/README.md

```diff
@@ -0,0 +1,35 @@
+# RainCatcher Angular.js Extensions
+
+Set of angularjs directives to extend existing UI components.
```

Comment:
it seems that this only allows to extend workorder UI so maybe it would be good to specify that? For me, this sentence is telling me that I can extend any UI components, not just workorders, with this module.

Reply:
It is now, but this will be used to extend overall ui components like step parameters in the near future.  Having separate modules for that seems wrong - I created that intentionally to make sure that team will be aware of this module and use it to satisfy any external modifications.